dot/rpc/subscription/listeners.go

```diff
@@ -286,3 +286,35 @@ func (l *RuntimeVersionListener) Listen() {
 // Stop to runtimeVersionListener not implemented yet because the listener
 // does not need to be stoped
 func (l *RuntimeVersionListener) Stop() {}
+
+// GrandpaJustificationListener struct has the finalisedCh and the context to stop the goroutines
+type GrandpaJustificationListener struct {
+	cancel context.CancelFunc
+	ctx    context.Context
+	wsconn *WSConn
+	subID  uint32
+
+	finalisedChID byte
+	finalisedCh   chan *types.FinalisationInfo
+}
+
+const grandpaJustifications = "grandpa_justifications"
+
+// Listen will start goroutines that listen to the finaised blocks
+func (g *GrandpaJustificationListener) Listen() {
+	// listen for finalised headers
+	go func() {
+		for {
+			select {
+			case info := <-g.finalisedCh:
+				hash := info.Header.Hash().String()
+				g.wsconn.safeSend(newSubscriptionResponse(grandpaJustifications, g.subID, hash))
```

Comment:
this doesn't send the grandpa justification, just the hash. the spec says it returns the block hash and the justification: https://github.com/w3f/PSPs/blob/master/PSPs/drafts/psp-6.md#173-grandpa_subscribejustifications-pubsub although it is very vague, I will try to clarify this

Reply:
Ok!